tf-psa-crypto/drivers/builtin/src/md5.c

```diff
@@ -58,7 +58,6 @@ int mbedtls_md5_starts(mbedtls_md5_context *ctx)
     return 0;
 }
 
-#if !defined(MBEDTLS_MD5_PROCESS_ALT)
 int mbedtls_internal_md5_process(mbedtls_md5_context *ctx,
```

Comment:
```suggestion
static int mbedtls_internal_md5_process(mbedtls_md5_context *ctx,
```
?
Same for all such functions documented as "internal"?

Reply:
You mean to remove from header file and make static for all the internal functions in `md5.h`, `ripemd160.h, `sha256.h` and so on?

Reply:
I mean to do what you have done for `mbedtls_des_setkey` (remove the prototype from the header and make it static) for the functions impacted by this PR that as `mbedtls_des_setkey` are "flagged" as internal either by their name (like `mbedtls_internal_md5_process`) or by their prototype. For the ones that are not explicitly flagged as internal (like  `mbedtls_des_crypt_ecb()`) it is okay to keep them as they are I'd say.

Reply:
Sorry me saying "remove the header file" is completely terrible wording. I meant to write "remove the protoype from the header file". I will do that this morning.